### PR TITLE
Changed split of command

### PR DESCRIPTION
**Pull request by ykz89**

### CODE REVIEW

File: Spotify.py [hunk 1]
```diff
@@ -102,31 +102,33 @@ def handle_message(channel, user, message):
...
-                    params = int(params)
-                    if 5 > params > 1 and Spotify.check_user(user):
-                        Spotify.execute_command("del {}").format(params)
-                        USERS[user] = datetime.datetime.now() + datetime.timedelta(minutes=config.WAIT_TIME)
-                    else:
-                        return "Already skipped or deleted a song from the queue in the last {} minutes. " \
-                               "Try again later".format(config.WAIT_TIME)
-                except ValueError:
-                    return "Please choose a number between 1 - {}".format(str(config.QUEUE_LENGTH))
+                if params is not None:
```
Comment (codeplexnl) [inline Spotify.py]: What happens if params is None?

File: Spotify.py [hunk 2]
```diff
@@ -102,31 +102,33 @@ def handle_message(channel, user, message):
...
-                param_first, param_second = params.split(None, 1)
-                try:
-                    param_first = int(param_first)
-
-                    if param_second.__eq__(''):
-                        return random.randint(1, int(param_first))
-                    elif param_second.isdigit():
-                        return random.randint(param_first, int(param_second))
-                    else:
+                if params is not None:
```
Comment (codeplexnl) [inline Spotify.py]: What happens if params is None?

**Review by codeplexnl — CHANGES_REQUESTED** [review]

File: Spotify.py (hunk 1 above)
Comment (ykz89) [reply Spotify.py]: Changed, so it shows the error when params is None

File: Spotify.py (hunk 2 above)
Comment (ykz89) [reply Spotify.py]: Changed if split or conversion fails, returns random number 1-10

**Review by codeplexnl — APPROVED** [review]

**Pull request merged**